demo-shell/src/app/app.routes.ts

```diff
@@ -85,66 +85,61 @@ export const appRoutes: Routes = [
     },
     { path: 'preview/s/:id', component: SharedLinkViewComponent },
     {
-        path: 'breadcrumb',
-        canActivate: [AuthGuardEcm],
-        component: AppLayoutComponent,
-        loadChildren: 'app/components/breadcrumb-demo/breadcrumb-demo.module#AppBreadcrumbModule'
-    },
-    {
-        path: 'notifications',
+        path: '',
         component: AppLayoutComponent,
+        data: { reuse: true },
```

Comment:
this thing does not exist in the ADF

Reply:
Passing a constant data stop the rerendering of the applayout I am investigating it

Reply:
you are right is enough moving all under the right parent as I did reuse is a thing of my immagination :)